ddtrace/appsec/_iast/_ast/visitor.py

```diff
@@ -38,71 +40,71 @@ def _mark_avoid_convert_recursively(node):
 
 _ASPECTS_SPEC: Dict[Text, Any] = {
     "definitions_module": "ddtrace.appsec._iast._taint_tracking.aspects",
-    "alias_module": "ddtrace_aspects",
+    "alias_module": _PREFIX + "aspects",
     "functions": {
-        "StringIO": "ddtrace_aspects.stringio_aspect",
-        "BytesIO": "ddtrace_aspects.bytesio_aspect",
-        "str": "ddtrace_aspects.str_aspect",
-        "bytes": "ddtrace_aspects.bytes_aspect",
-        "bytearray": "ddtrace_aspects.bytearray_aspect",
-        "ddtrace_iast_flask_patch": "ddtrace_aspects.empty_func",  # To avoid recursion
+        "StringIO": _PREFIX + "aspects.stringio_aspect",
```

Comment:
what about move this to a function?
```
_build_iast_module_path(string):
    return _PREFIX + "aspect." + string
```

and
```
"StringIO": _build_iast_module_path("aspects.stringio_aspect"),
```

Reply:
I don't see the point. If this was more complicated I would see it, but it's just a concat (actually I am thinking of using `f"{_PREFIX}aspects.stringio_aspect"` but the `+` is more readable.